spec/intergration/posts_index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#index', type: :feature do
+  describe 'Post' do
+    before(:all) do
+      @user = User.first!
+      @post1 = Post.first!
+      @post2 = Post.second!
+      @post3 = Post.third!
+    end
+
+    it "shows user's profile picture" do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_css('img[src*="https://dummyimage.com/600x400/000/fff"]')
+    end
+
+    it 'shows the users username' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content('frank')
+    end
+
+    it 'shows number of posts of user has written' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content('Number of posts: 3')
+    end
+
+    it 'shows posts title' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content('Ruby and Rails 1')
+    end
+
+    it 'can see some of the post body' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content 'this is first post about Ruby and Rails! 1'
+    end
+
+    it 'can see the first comment on a post' do
+      visit(user_posts_path(@user.id))
+      expect(page).to have_content 'Absolutely amazing'
+    end
+
```

Comment:
- [ ] Please add a test that asserts you can see how many comments a post has.
- [ ] Please add a test that asserts you can see how many likes a post has.
- [ ] Please add a test that asserts you can see a section for pagination if there are more posts than fit on the view.
Because you forgot to add them.

![image](https://user-images.githubusercontent.com/68796232/186076234-19ce6c82-36e7-44a9-9ce1-7227a114a780.png)